angular_components/1-base/lib/src/settings/settings.dart

```diff
@@ -2,8 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'package:angular2/angular2.dart';
-import 'package:components_codelab/src/lottery/lottery.dart';
+import 'package:angular/angular.dart';
+import '../lottery/lottery.dart';
```

Comment:
There's no official recommendation for imports within the same package. Personally, I tend to use relative ones, though I know others dislike them.